Automattic/wp-calypso
# Jetpack Manage: Update the Preview Pane with the feature tabs bar and content with the Jetpack feature components

Resolves: https://github.com/Automattic/jetpack-manage/issues/316
Resolves: https://github.com/Automattic/jetpack-manage/issues/293

## Proposed Changes

This PR adds the feature tabs bar to the Preview Pane:

![image](https://github.com/Automattic/wp-calypso/assets/9832440/f3147d90-4d5f-43e3-a85a-27edb7e5f012)


I designed the system to be agnostic from the context, so the features are assigned from an external component. For this context, Manage, I create the `JetpackPreviewPane` that wraps `SitePreviewPane` component to feed it with the features of the context.

- JetpackPreviewPane
    - Configures these Jetpack features with `id`, `tab`, `preview`
        - `JetpackBoostPreview`
        - `JetpackBackupPreview`
        - `JetpackStatsPreview`
        - `JetpackMonitorPreview`
    - Render:
        -  `SitePreviewPane` **props:** `site`, `features`, `closeSitePreviewPane`
            - It will render the Preview Pane: header, tabs, and the content (preview).

## Testing Instructions

- Check the code and how the Preview Pane with multiple features works.
- Test the UI/UX, click on the different tabs, and, with the Preview Pane opened, click on another site.

## Pre-merge Checklist
